src/lsp/cobol_ptree/terms.ml

```diff
@@ -694,9 +892,64 @@ module FMT = struct
     | DataAddress i -> fmt "ADDRESS@ OF@ %a" ppf pp_ident i
     | ProgAddress i -> fmt "ADDRESS@ OF@ PROGRAM@ %a" ppf pp_term i
 
-  and pp_inline_call ppf { call_fun; call_args } =
-    fmt "FUNCTION@ %a@ @[<1>(%a)@]" ppf pp_name' call_fun
-      (list ~sep:comma pp_effective_arg) call_args
+  and pp_inline_call ppf f =
+    match f with
+    | CallFunc { call_fun; call_args } ->
+      fmt "FUNCTION@ %a@ @[<1>(%a)@]" ppf pp_name' call_fun
+        (list ~sep:comma pp_effective_arg) call_args
```

Comment:
Just noticed this: I think `~sep:comma` should be `~sep:sp` instead (commas are like space separators).